apps/video/index.html

```diff
@@ -64,59 +64,79 @@ <h1 id="thumbnails-number-selected"></h1>
 
         <!-- This element gets positioned over different view elements -->
         <!-- It gets styled differently depending on what it is over -->
-        <ul id="thumbnails"></ul> <!-- Thumbnails inserted here -->
+        <!-- The real thumbnails list is actually inside this ul. We need
+             to remove the semantics of this list to be less verbose, in the
+             screen reader is on. -->
+        <ul role="presentation" id="thumbnails"></ul> <!-- Thumbnails inserted here -->
```

Comment:
I'm thinking the comment is supposed to say "The real thumbnails list is actually inside this ul. We need to remove the semantics of this list if the screen reader is on." Is that correct?